core/tests/protractor_desktop/topicAndStoryEditor.js

```diff
@@ -295,6 +298,21 @@ describe('Chapter editor functionality', function() {
     storyEditorPage.saveStory('First save');
   });
 
+  it(
+    'should check presence of skillreview RTE element in exploration ' +
+    'linked to story', function() {
+      browser.get('/create/' + dummyExplorationIds[0]);
+      waitFor.pageToFullyLoad();
+      explorationEditorMainTab.setContent(function(richTextEditor) {
+        richTextEditor.addRteComponent('Skillreview', 'Description');
+      });
+      explorationEditorPage.navigateToPreviewTab();
+      explorationPlayerPage.expectContentToMatch(function(richTextChecker) {
+        richTextChecker.readRteComponent(
+          'Skillreview', 'Description', forms.toRichText('material1'));
```

Comment:
Should check that clicking on the component goes to the right place.

Reply:
Yeah, this is done. In Skillreview/protractor.js, the `expectComponentDetailsToMatch()` function actually does click on the element and verifies that the review material is the expected one (which is passed in as 'material1' here).